recipes/statsmodels/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "statsmodels" %}
+{% set version = "0.8.0rc1" %}
```

Comment:
We have yet reach a consensus on how we do rc/dev releases. ( https://github.com/conda-forge/conda-forge.github.io/issues/147 ) Though I think we are all interested in that possibility. Would it be alright if we switch to a non-dev release?